_board/bast_wifi.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: download
+board_id: "bast_wifi"
+title: "bast_wifi Download"
+name: "bast_wifi"
```

Comment:
Could you change the name and title from `bast_wifi` to `Bast WiFi` since that's how it will appear on the website.